src/pages/auth/SignIn.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from 'react-redux'
+import {push as routerPush} from 'react-router-redux';
+
+// import { Link } from "react-router-dom";
+import { signIn } from "../../redux/actions/authActions";
+
+import {toastr} from 'react-redux-toastr'
+
+import {
+  Button,
+  Card,
+  CardBody,
+  // Form,
+  FormGroup,
+  // Label,
+  // Input,
+  // CustomInput,
+  Spinner,
+} from "reactstrap";
+
+import {
+  AvForm,
+  AvField
+} from "availity-reactstrap-validation";
+
+
+// import avatar from "../../assets/img/avatars/avatar.jpg";
+
+const SignIn = () => {
+  // console.log('ok')
+  const dispatch = useDispatch();
+
+  const [email, setEmail] = useState('Keeley50@yahoo.com');
+  const [password, setPass] = useState('');
+
+  const {authLoading, isAuthenticated, authUser} = useSelector(state => {
+    // console.log(state);
+    return state.auth
+  });
+
+  function handleEmailChange(e) { setEmail(e.target.value) }

```

Comment:
why do you write function declaration here if you are using const functions everywhere?!!